aur/packages

```diff
@@ -39,6 +39,7 @@ rakarrack-plus
 rezonateur
 rtcqs
 sfzlint
+simplescreenrecorder
```

Comment:
We can only update packages in the AUR, for which we can ad the `osamc` account as a (co-)maintainer, which is not the case here. This line should be removed.